vilebot/src/main/java/com/oldterns/vilebot/Vilebot.java

```diff
@@ -113,20 +121,73 @@ public static void main( String[] args )
             String ircServerAddress = cfg.get( "ircServerAddress" + i );
             int ircPort = Integer.parseInt( cfg.get( "ircPort" + i ) );
             String ircChannel = cfg.get( "ircChannel" + i );
+            String listenerCsvString = cfg.get( "listeners" + i );
```

Comment:
This additional cfg entry should also be added in `vilebot/cfg/vilebot.conf.example` (ie. `listeners1=all`).